framework/db/BaseActiveRecord.php

```diff
@@ -1286,4 +1286,41 @@ public static function isPrimaryKey($keys)
 			return false;
 		}
 	}
+
+	/**
+	 * Returns the text label for the specified attribute.
+	 * If the attribute looks like `relatedModel.attribute`, then the attribute will be received from the related model.
+	 * @param string $attribute the attribute name
+	 * @return string the attribute label
+	 * @see generateAttributeLabel()
+	 * @see attributeLabels()
+	 */
+	public function getAttributeLabel($attribute)
+	{
+		$labels = $this->attributeLabels();
+		if (isset($labels[$attribute])) {
+			return ($labels[$attribute]);
+		} elseif(strpos($attribute, '.')) {
```

Comment:
I'm poking fun @djagya, who well knows how much I dislike else-after-return. 👅

If you remove the `else` from this line, move the remainder of the line to the next line, you can remove the `}` on line 1322 and the block moves left 4 spaces. nicer and easier to read. the extra `else` and indent serves no purpose. 🌽